.gitignore

```diff
@@ -140,9 +140,6 @@ junit/
 raw-test-output/
 *results.xml
 
-# Virtualenv on tails
-.venv
```

Comment:
we should keep this because this is where we keep the virtualenv on tails (admins/journalists use git checkouts on tails)

Reply:
It's redundant; `.venv*` is present above.